### PR TITLE
initial number is 0

### PR DESCRIPTION
**Pull request by AliUmman**
I think initial number should be not be grater than zero

### CODE REVIEW

**Review by AyseUmman — COMMENTED** [review]
I agree with u, 0 is the true option

File: HelloWorld.java [hunk 1]
```diff
@@ -4,7 +4,7 @@ public static void main(String[] args) {
         System.out.println("Hello, World");
         
         int maxNum = 100;
-        for(int i = 3; i < maxNum; i++)
```
Comment (AyseUmman) [inline HelloWorld.java:7]: this line should removed

Comment (AliUmman) [reply HelloWorld.java:7]: lets ask to Veli

Comment (AyseUmman) [reply HelloWorld.java:7]: he also said 0

**Review by AyseUmman — APPROVED** [review]

File: HelloWorld.java (hunk 1 above)
Comment (AyseUmman) [reply HelloWorld.java:7]: sorry, 3 Veli said

**Pull request merged**